sentimentanalysis/analyzer.py

```diff
@@ -0,0 +1,502 @@
+import sys
+import math
+import re
+import string
+from itertools import product
+import nltk.data
```

Comment:
BIG problem. ```nltk``` relies on ```sqlite``` which I'm fairly sure Heroku does not support. This might make Heroku deployment a no-go we we are going to use NLTK.